code42_connector.py

```diff
@@ -40,6 +64,17 @@ def _convert_to_obj_list(scalar_list, sub_object_key="item"):
     return [{sub_object_key: item} for item in scalar_list]
 
 
+def add_eq_filter(filters, value, filter_class):
+    return filters.append(filter_class.eq(value))
+
+
+def is_default_dict(_dict):
```

Comment:
this gets shared between alerts and file events - you can use it to check to see if all the params were left blank

Reply:
`has_values` might be a better name.

Reply:
Nevermind, now I see what it is called this